node/block_processor/status.go

```diff
@@ -11,7 +11,7 @@ type blockExecStatus struct {
 	startTime, endTime time.Time
 	height             int64
 	txIDs              []ktypes.Hash
-	txStatus           map[string]bool
+	txStatus           map[ktypes.Hash]bool
```

Comment:
One of the nice things about defining a `Hash` type as a fixed length array `[32]byte` is that it is `comparable` and can be a map key.  It's also nice to ensure we always encode to the same length hex chars, which we didn't do before (see old `display` tests where we had "hashes" displayed of just a few chars :exploding_head: )